data.yaml

```diff
@@ -26,3 +26,9 @@
   links:
     - lang: "DE"
       url: "https://bank.dkb.de/fragen-antworten/wie-melde-ich-eine-phishing-mail-oder-eine-verdaechtige-sms"
+
+- name: IONOS
+  address: "--"
```

Comment:
Can you please change this to `address: ""` then it gets ignored by the script and we wont generate an link with `mailto:--`